### ISSUE TITLE
[Keyboard Navigation - Pane]: "Focus does not move to Feature Showcases" dropdown present under 'Grafana Play Home' making it keyboard inaccessible.

### ISSUE THREAD

**Issue by natellium** [grafana/grafana#66250]
### User Experience: 
Keyboard-dependent users will not be able to access the controls and will not be able to perform the required action on the controls if interactive controls present on any page or pane are not keyboard accessible, hence fail to use their functionality. 

### Repro Steps:
Open #URL: https://play.grafana.org/. Sign in with valid credentials. "Home" page appears. Navigate the Feature Showcases pane. Verify the issue.

### Actual Result: Issue: 
"Feature Showcases" dropdown and the controls i.e. (More, Duplicate, copy...etc.) are not accessible through keyboard under 'Grafana Play Home' screen as focus does not move to it.
Observation: While navigating through keyboard, focus does not move to "Feature Showcases" dropdown, so it is not accessible via keyboard. 

### Expected Result:
 The "Feature Showcases" dropdown and the controls i.e. (More, Duplicate, copy...etc.) should receive keyboard focus and be accessible through keyboard.
Note: The same issue exists with "Data Source Demos and What's new" dropdown under same page. Same issue exists with Share Dashboard panel->Link Same issue exists with Share Dashboard panel->Export. Same issue exists with same controls throughout the application. Same issue exists with explore->Inspector

### Severity
`SEV2`

**Comment by natellium** [comment]
I believe this may have been fixed by the new panel header

**Comment by ckbedwell** [comment]
I've checked and whilst technically the options are now keyboard accessible it isn't intuitive as you have to focus the panel first and then move focus backwards to access the menu items, so we still fail on [wcag/2.4.3](https://www.w3.org/TR/WCAG21/#focus-order).




https://github.com/grafana/grafana/assets/6906380/5f6003f4-2b62-4405-b882-5ea59106ed48

**Comment by grafanabot** [comment]
Thanks for creating this issue! It looks like this has already been reported by another user. We’ve closed this in favor of the existing one. Please consider adding any details you think is missing to that issue.

To avoid having your issue closed in the future, please read our [CONTRIBUTING](https://github.com/grafana/grafana/blob/main/CONTRIBUTING.md) guidelines.

Happy graphing!
